docker_templates/templates/docker_images/create_drcsim_image.Dockerfile.em

```diff
@@ -18,24 +18,46 @@
     os_name=os_name,
     os_code_name=os_code_name,
 ))@
+@{
+packages = [
+    'dirmngr',
+    'gnupg2',
+    'lsb-release'
+]
+if 'upstream_packages' in locals():
```

Comment:
Would you also like to make the line 27-32 a imported template in its own right, as the snippet duplicated a lot.
Or is that getting to your issue with EmPy?

Reply:
Yeah that's getting to the scope issue I was facing, as suggested by Tully above we would need a more complex approach than templates to achieve this